docs/en/getting-started.md

```diff
@@ -1 +1,154 @@
+
+### How to wrap a component
+
+As mentioned above, the wrapper is the core element that allows for efficient interaction with a component. The flow goes as following:
+
+```js
+// Import the mount() method from the test utils
+// and the component you want to test
+
+import { mount } from 'vue-test-utils'
+import Counter from './counter'
+
+// Now mount the component and you have the wrapper
+const wrapper = mount(Counter)
+
+// To inspect the wrapper deeper just log it to the console
+// and your adventure with the vue-test-utils begins
+console.log(wrapper)
+```
+
+We already took the first hurdle, and luckily it was an easy one. So, let's put the wrapper to good use.
+
+
+### Test rendered HTML output of the component
+
+As a first spec it would be nice to verify that the rendered HTML output of the component looks like expected.
+
+```js
+// For getting the html output, the wrapper provides
+// a html() method for you.
+
+describe('Counter', () => {
+  it('renders the correct markup', () => {
+    expect(wrapper.html()).to.equal('<div>0<button>Increment</button></div>')
+  })
+
+  // it's also easy to check for the existence of elements
+  it('has a button', () => {
+    expect(wrapper.contains('button')).to.equal(true)
+  })
+})
+```
+
+That was easy as well, so let's step up the game.
+
+### Component data
+
+Changing the data of the component can be quite useful for efficient testing. The method `setData({...})` is ment for changing the data and `data()` returns the current data object of the component.
+It may be useful to change the data accordingly for a whole group of specs, so `beforeEach()` could be a good place for that:
+
+```js
+
+describe('Data interactions', () => {
+  beforeEach(() => {
+    wrapper.setData({ count: 10 })
+  })
+
+  it('should be set to 10', () => {
+    expect(wrapper.data().count).to.equal(10)
+  })
+})
+
+```
+
+At this point you should also know that you can interact with the vue instance itself as well using the `vm` key.
+
+```js
+  wrapper.vm.$data
+  // is equal to
+  wrapper.data()
+```
+
+### Interactions
+
+This section will introduce two important methods of the wrapper object.
+`find()` to find an HTML element inside your component's HTML output and `trigger()` will allow for firing general events.
+
+```js
+
+describe('Trigger an event', () => {
+  it('button should increment the count', () => {
+    expect(wrapper.data().count).to.equal(0)
+    const button = wrapper.find('button')
+    button.trigger('click')
+    expect(wrapper.data().count).to.equal(1)
+  })
+})
+
+```
+
+### Handle async DOM updates
+
+Vue updates the DOM based on an internal `tick` to prevent unnecessary rerenders if a bunch of data gets changed. That's why `Vue.nextTick(...)` allows you to verify that DOM changes have been applied correctly and to pass in a callback to run right after that. Luckily this async DOM update is taked care of by the `.html()` method.
+
+```js
+
+describe('DOM updates', () => {
```

Comment:
vue-test-utils has an `update` method that is called inside setData.  `html` doesn't update the DOM.

So we should use `update` as an example instead. Here's the test- https://github.com/vuejs/vue-test-utils/blob/master/test/integration/specs/mount/Wrapper/update.spec.js#L6 . There's probably a better example to use than that though